tests/marine/test_marine.py

```diff
@@ -2,85 +2,252 @@
 Note: in order to run the tests, you must put libmarine.so next to the marine_fixtures.py file
 """
 import pytest
+from typing import List, Dict
+from marine import Marine
+
+
+def test_arp_filter_and_parse(
```

Comment:
All of the `filter_and_parse` tests look the same - have you considered turning them into a single parameterized test?

Reply:
Don't forget to use ids :)